tests/_data/structure.sql

```diff
@@ -0,0 +1,8 @@
+CREATE DATABASE IF NOT EXISTS phalcon CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+create table articles
+(
+    id int auto_increment,
+    title varchar(255) null,
+    constraint test_pk
```

Comment:
I thought that primary key can only be named PRIMARY.

Reply:
I think both is possible. Changed in the dev branch to the common format.